internal/controller/observability_reconciler.go

```diff
@@ -400,34 +344,19 @@ func (r *ObservabilityReconciler) Reconcile(baseCtx context.Context, _ []control
 		return o.GroupVersionKind().GroupKind() == machinery.GatewayClassGroupKind
 	})
 
-	wantedSMs := []monitoringv1.ServiceMonitor{
-		*kOpMonitor,
-		*dnsOpMonitor,
-		*authOpMonitor,
-		*limitOpMonitor,
```

Comment:
is it correct to remove these from the wanted list?

Reply:
yep, I removed the logic for cleaning up serviceMonitors [here](https://github.com/Kuadrant/kuadrant-operator/pull/1352/files#diff-f501d6543c97ef40ce2c7521c60283cc4921ae7af0c4213de18c9ad0b02ac79bL445) so these are no longer needed.